eng/packaging.targets

```diff
@@ -288,4 +288,19 @@
     <Error Condition="'$(ServicingVersion)' == '0'" Text="ServicingVersion is set to 0 and it should be an increment of the patch version from the last released package." />
   </Target>
 
+  <ItemDefinitionGroup>
+    <TargetPathWithTargetPlatformMoniker>
+      <GeneratePackageOnBuild>$(GeneratePackageOnBuild)</GeneratePackageOnBuild>
+    </TargetPathWithTargetPlatformMoniker>
+  </ItemDefinitionGroup>
+
+  <Target Name="ValidateBuiltPackageInServicing" 
```

Comment:
Consider refining this target's name as the `ValidateServicingVersionIsPropertlySet` target also performs servicing validation. Alternatively consider merging both targets together.